CompanyEmployees.Core.Domain/Entities/Attendance.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace CompanyEmployees.Core.Domain.Entities
+{
+    public class Attendance
+    {
+        [Column("AttendanceId")]
+        public Guid Id { get; set; }
+
+        [Required]
+        [ForeignKey(nameof(Employee))]
+        public Guid EmployeeId { get; set; }
+
+        [Required]
+        public DateTime CheckInTime { get; set; }
+
+        public DateTime? CheckOutTime { get; set; }
+
+        [Required]
+        [MaxLength(10, ErrorMessage = "Maximum length for the Status is 10 characters.")]
+        public string? Status { get; set; } // Present, Absent, Late, etc.
+
+        [MaxLength(200, ErrorMessage = "Maximum length for the Notes is 200 characters.")]
+        public string? Notes { get; set; }
+
+        public Employee? Employee { get; set; }
```

Comment:
The `Employee` navigation property is marked as nullable, but since `EmployeeId` is required and the relationship appears to be mandatory, consider making `Employee` non-nullable for better clarity and consistency.

**Suggested change**
```suggestion
public Employee Employee { get; set; }
```